PWGMM/Mult/TableProducer/trackPropagation.cxx

```diff
@@ -227,10 +190,12 @@ struct AmbiguousTrackPropagation {
     if (atracks.size() == 0) {
       return;
     }
+    initCCDB(bcs.begin());
 
     // Only on DCAxy
     float dcaInfo;
```

Comment:
This is not used further.

Reply:
Actually it is, it fills the value of Bz (mag field for MFT)